toolvoice/scripts/inference/openai_realtime_infer.py

```
@@ -0,0 +1,386 @@
+def parse_tool_cell(cell: str) -> Union[List[Any], Dict[str, Any], List[str]]:
+    if not cell:
+        return []
+    s = cell.strip()
+
+    if s.startswith("@"):
+        return read_json_from_file(s[1:].strip())
+
+    try:
+        return json.loads(s)
+    except Exception:
+        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)